bundles/org.openhab.binding.haassohnpelletstove/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -11,7 +11,7 @@
 			optional
 			WLAN-Modul. More information can be found here: https://www.haassohn.com/de/ihr-plus/WLAN-Funktion. It allows
 			to power on/off the stove as well as receiving different operation information about the stove.</description>
-
+		<semantic-equipment-tag>Oven</semantic-equipment-tag>
```

Comment:
I don't think it's that kind of oven, i.e. a `WhiteGood`: https://www.haassohn.com/produkte-en/pellet-stoves.

Reply:
Ok

Reply:
Done.